app/views/admin/AdministerUserModal.coffee

```diff
@@ -261,7 +261,10 @@ module.exports = class AdministerUserModal extends ModalView
           @renderSelectors('#'+prepaidId)
         return
 
-  userIsSchoolAdmin: -> @user.isSchoolAdmin()
+  userIsSchoolAdmin: ->
+    console.log(@user.isSchoolAdmin())
```

Comment:
Do we need these logs?

Reply:
Ooops, nope! Left over - thank you for catching!